main.go

```diff
@@ -1,30 +1,60 @@
 package main
 
 import (
-    "os"
-    "fmt"
     "flag"
-    "./imagetocsv"
-    "./learning"
-    "./web"
+    "fmt"
+    "github.com/AKeizer/TeamGandalf/imagetocsv"
+    "github.com/AKeizer/TeamGandalf/learning"
+    "github.com/AKeizer/TeamGandalf/web"
+    "os"
+    "os/exec"
+    "strings"
 )
 
 func helpText() {
     fmt.Printf("Usage: %s [flags] out_file in_file(s)\n", os.Args[0])
 }
 
+func buildImageConvert(output string, inputs []string) (*exec.Cmd, error) {
+    path, err := exec.LookPath("imagecsv")
+    if err != nil {
+        return nil, err
+    }
+    return exec.Command(path, output, strings.Join(inputs, " ")), nil
+}
+
+func runImageConvert(imageConverter *exec.Cmd) error {
+    return imageConverter.Run()
+}
+
+func buildMLExec(training bool, inputFile *os.File) (*exec.Cmd, error) {
+    path, err := exec.LookPath("learning")
+    if err != nil {
+        return nil, err
+    }
+    return exec.Command(path, inputFile.Name()), nil
+}
+
+func runML(machine *exec.Cmd) error {
+    return machine.Run()
+}
+
 func main() {
-    totalpixels := 400
+    // Set up flags
+    train := flag.Bool("train", false, "Use the input files as training")
+    help := flag.Bool("h", false, "Print help text")
+    isWeb := flag.Bool("web", false, "Setup webserver")
 
-    var train = flag.Bool("train", false, "If we should use the inputs as training")
-    var help = flag.Bool("h", false, "Should we display the help information")
-    var isWeb = flag.Bool("web", false, "Setup webserver")
-    flag.Parse()
     if len(os.Args) < 2 && !*isWeb {
         helpText()
         os.Exit(1)
     }
 
+    if len(os.Args) < 2 {
```

Comment:
Not sure i understand why this is needed when there's a check just above-- If I understand correctly, this will just print the help text and ext even if it is web.